src/javascript/_common/base/client_base.js

```diff
@@ -403,6 +403,13 @@ const ClientBase = (() => {
         return options_blocked_countries.includes(country);
     };
 
+    const isMalta = () => {
+        const malta_countries = ['at', 'be', 'bg', 'cy', 'cz', 'dk', 'ee', 'fi', 'hr', 'hu', 'ie', 'lt', 'lv', 'nl', 'pl', 'pt', 'ro', 'se', 'si', 'sk'];
```

Comment:
in `landing_company` response, you can check for `gaming_company` and then see if the shorcode inside that object contains `shortcode: "malta"`.  -> `landing_company.gaming_company.shortcode` is the key here. 

This works for Virtual clients too.

Reply:
sure, work on that